### PR TITLE
feat(maintner): speed up GetIssue function 

### PR DESCRIPTION
**Pull request by orthros**
by not iterating each Issue in the corpus

### CODE REVIEW

**Comment by orthros** [comment]
This PR reduces time on GetIssue requests significantly. I'm seeing requests in prod taking `~700ms` to be reduced to `~130ms`

File: drghs-worker/maintnerd/api/v1beta1/issue_service.go [hunk 1]
```diff
@@ -236,16 +240,17 @@ func (s *IssueServiceV1) GetIssue(ctx context.Context, r *drghs_v1.GetIssueReque
...
-		return repo.ForeachIssue(func(issue *maintner.GitHubIssue) error {
-			if r.Name == getIssueName(repo, issue) && !issue.NotExist {
-				re, err := makeIssuePB(issue, repo.ID(), r.Comments, r.Reviews, r.FieldMask)
-				if err != nil {
-					return err
-				}
-				issueResp = re
-			}
+		issue := repo.GetIssue(issueID)
+		if issue == nil {
```
Comment (jmdobry) [inline drghs-worker/maintnerd/api/v1beta1/issue_service.go:244]: don't also need to check if the issue is NotExist?

File: drghs-worker/maintnerd/api/v1beta1/issue_service_test.go [hunk 2]
```diff
@@ -161,3 +161,21 @@ func TestIssueFilters(t *testing.T) {
 		}
 	}
 }
+
+func TestGetIsueID(t *testing.T) {
```
Comment (jmdobry) [inline drghs-worker/maintnerd/api/v1beta1/issue_service_test.go:165]: s/`TestGetIsueID`/`TestGetIssueID`

File: drghs-worker/maintnerd/api/v1beta1/issue_service.go (hunk 1 above)
Comment (orthros) [reply drghs-worker/maintnerd/api/v1beta1/issue_service.go:244]: Yeah.  @BrennaEpp and I caught that during a review earlier today. Will fix.

**Pull request merged**